woocommerce/woocommerce
# Use Emogrifier composer package

### All Submissions:






### Changes proposed in this Pull Request:



Now we can use Emogrifier as a composer package, since now we require PHP 7.0+ (Emogrifier requires 5.6+), and also we work with composer packages :)

### How to test the changes in this Pull Request:

1. Apply this patch.
2. Enable HTML or Multipart emails in the "New order" email: "wp-admin/admin.php?page=wc-settings&tab=email&section=wc_email_new_order"
3. Place an new order to trigger the "New order" email.
4. Check the email's source see the HTML, all CSS must be inline, without any `<style type="text/css"></style>` tag.

### Other information:




### Changelog entry

> Dev - Includes Emogrifier composer package instead of including into `includes/libraries`.
